oca/decorators/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from .foreach import foreach
+
+
+__all__ = [
```

Comment:
@lasley You can leave this file empty.

Reply:
This is required in order to make the `foreach` method inside of the `foreach` file directly importable. 

Otherwise usage is:

```
from oca.decorators.foreach import foreach
```

instead of

```
from oca.decorators import foreach
```

Do you have a better way to solve this?

Reply:
:+1:

Reply:
Do we really want one file per decorator? 😲

Reply:
I generally prefer deeper file hierarchies with smaller files. I've found that if I don't do this type of thing from the get-go I end up in a situation where the files are thousands of lines deep, I spend a bunch of time trying to categorize things, then just lose things anyways.

I'm open to consideration here though. You would prefer to categorize/group somehow?

Reply:
@lasley IMO Your way of organizing your code is safe.